.github/workflows/build-and-release.yml

```diff
@@ -218,6 +218,20 @@ jobs:
         # Remove conflicting brew items
         brew remove --ignore-dependencies libpng brotli harfbuzz
 
+    - name: "Download openssl x86_64"
```

Comment:
Following might be safer. I didn't confirmed yet.
```yaml
- name: "Fetch openssl x86_64 bottle via brew"
  continue-on-error: true
  run: |
    set -euxo pipefail
    brew update
    brew fetch --force-bottle --bottle-tag=x86_64_sonoma openssl@3
    TAR=$(ls "$(brew --cache)"/**/*openssl@3*--x86_64_sonoma*.tar.gz | head -n1)
    tar -xzf "$TAR"
    ln -sfn ./openssl@3/3.* ./openssl_3
```

Reply:
https://github.com/nijigenerate/nijigenerate/pull/126/commits/59101633bf4134ea78efa47c5755200817891246 Looks like it works, grep tar.gz and extract it
https://github.com/r888800009/nijigenerate/actions/runs/17534997076/job/49796891483